cmd/mtaBuild.go

```diff
@@ -192,12 +194,13 @@ func getMarJarName(config mtaBuildOptions) string {
 }
 
 func addNpmBinToPath(e execRunner) error {
-	path := "./node_modules/.bin"
+	dir, _ := os.Getwd()
+	newPath := path.Join(dir, "node_modules", ".bin")
 	oldPath := os.Getenv("PATH")
 	if len(oldPath) > 0 {
-		path = path + ":" + oldPath
+		newPath = newPath + ":" + oldPath
 	}
-	e.SetEnv(append(os.Environ(), "PATH="+path))
+	e.SetEnv([]string{"PATH=" + newPath})
```

Comment:
Does this work? As far as I understood the issue it is caused by using an ExecRunner here which apparently (?) does not spawn a new process and works with the current environment. So I would expect something like os.Setenv() inside `pkg/command/command.go` in method `RunExecutable`. But maybe I misunderstood something ...

Reply:
This is just an upfront change that fixes the fact that we add the `os.Environ()` two times. The issue described in #1223 still persists.

Reply:
You mean because `os.Environ()` is already prepended in `RunExecutable()` and `RunShell()` via `appendEnvironment()`?

Reply:
Ah, followed a link from a notification and only now saw you linked the spot already. So I understood correctly.